javascript/fluent.js

```diff
@@ -125,16 +125,29 @@
 				} else {
 					label.attr('title', 'Using default locale value');
 				}
+			},
+			/**
+			 * The JSON in the data attribute will be encoded, so this is a bit of a hack to decode it before parsing
```

Comment:
I'm guessing it's [`Convert::raw2xml`](https://github.com/silverstripe/silverstripe-framework/blob/3/core/Convert.php#L99-L106) that makes it so, called from `FormField::getAttributesHTML`